Adyen/UI/Form/Items/Value Pickers/Abstract/BaseFormPickerItem.swift

```diff
@@ -0,0 +1,57 @@
+//
+// Copyright (c) 2021 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import UIKit
+
+/// Picker item identifier.
+/// :nodoc:
+public protocol PickerElement: Equatable, CustomStringConvertible {
+
+    /// Picker item identifier.
+    /// :nodoc:
+    var identifier: String { get }
+}
+
+/// :nodoc:
+public struct BasePickerElement<T: CustomStringConvertible>: PickerElement {
+
+    /// Picker item identifier.
+    /// :nodoc:
+    public var identifier: String
+
+    /// Picker item value.
+    /// :nodoc:
+    public var item: T
+
+    public static func == (lhs: Self, rhs: Self) -> Bool {
+        lhs.identifier == rhs.identifier
+    }
+
+    public var description: String { item.description }
+
+}
+
+/// Describes a picker item.
+open class BaseFormValuePickerItem<T: CustomStringConvertible>: FormValueItem<BasePickerElement<T>, FormTextItemStyle>,
+    InputViewRequiringFormItem {
+
+    /// The complete list of selectable values.
+    internal let selectableValues: [BasePickerElement<T>]
+
+    /// The title of the item.
+    public var title: String?
```

Comment:
can we make it a constant and pass as a parameter to the initializer?

Reply:
This one is actually not used 🙊